cypress.json

```diff
@@ -1,10 +1,11 @@
 {
-  "chromeWebSecurity": false,
-  "baseUrl": "http://localhost:3000/",
-  "defaultCommandTimeout": 15000,
-  "viewportWidth": 1300,
-  "reporter": "junit",
-  "reporterOptions": {
-    "mochaFile": "cypress/results/cypress-report-[hash].xml"
-  }
-}
\ No newline at end of file
+	"chromeWebSecurity": false,
+	"baseUrl": "https://roundaround-stage.web.app/",
```

Comment:
So for this PR the tests will always point to stage?

Reply:
yes, but we can change it on a per branch basis. Also the local test use localhost, I set that up in package.json